apps/api/src/app/auth/usecases/login/login.usecase.ts

```diff
@@ -62,6 +63,19 @@ export class Login {
       throw new UnauthorizedException(`Incorrect email or password provided.`);
     }
 
+    if (process.env.INTERCOM_IDENTITY_VERIFICATION_SECRET_KEY && !user.servicesHashes?.intercom) {
+      const intercomSecretKey = process.env.INTERCOM_IDENTITY_VERIFICATION_SECRET_KEY as string;
+      const userHashForIntercom = createHash(intercomSecretKey, user._id);
+      await this.userRepository.update(
+        { _id: user._id },
+        {
+          $set: {
+            servicesHashes: { intercom: userHashForIntercom },
+          },
```

Comment:
```suggestion
          $set: {
            'servicesHashes.intercom': userHashForIntercom },
          },
```

The previous method could erase all the serviceHashes contents with this object